xdsl/dialects/riscv_scf.py

```diff
@@ -37,7 +37,7 @@ class YieldOp(IRDLOperation):
     traits = frozenset([IsTerminator()])
 
     def __init__(self, *operands: SSAValue | Operation):
-        super().__init__(operands=[SSAValue.get(operand) for operand in operands])
+        super().__init__(operands=[[SSAValue.get(operand) for operand in operands]])
```

Comment:
Why?

Reply:
each element in the operands array corresponds to an irdl operand definition. This class has one definition, so it should be a single-elemnt list. The one definition is variadic, hence the list.

Reply:
I see, thanks for explaining.